dash.el

```diff
@@ -1594,6 +1594,18 @@ elements of LIST.  Keys are compared by `equal'."
         (setq lists (-map 'cdr lists)))
       (nreverse result))))
 
+(defun -interleave-all (&rest lists)
+  "Return a new list containing an element of each of `LISTS' in turn, including all elements.  "
```

Comment:
IMO the first version of this docstring was a much clearer description. :) The only problem with it was that the first sentence wasn't a summary that fit entirely on a single line.  Apart from clarity, the current version has three technical issues:
1. It is longer than 80 characters, which emits an error during byte-compilation in Emacs 28+ and thus CI fails.
2. Metavariables are written unquoted, i.e. `` `LISTS' `` -> `LISTS`.
3. There is spurious trailing whitespace after the full stop.

Suggest writing something like the following:

```el
  "Zip all elements from each of LISTS into a new flat list.

That is, return a list comprising the first item from each of
LISTS in turn, followed by their second items, etc.

Continue interleaving until all elements from all LISTS are
included, skipping non-existing elements from shorter LISTS.
This is like `-interleave', but the interleaving continues until
all input elements are consumed, instead of stopping after one of
LISTS becomes empty."
```